setup.py

```diff
@@ -25,6 +27,108 @@
 except ImportError:
     pass
 
+
+def getfiles(path):
+    '''Retrieve file list within a directory
+
+    :param path: directory path
+    :type path: string
+    :returns: file list
+    :rtype: list
+    '''
+    for root, dirs, files in os.walk(path):
+        file_list = [os.path.join(root, file) for file in files]
+    return(file_list)
+
+
+def isdebianbased():
+    '''Check if it is a Debian based system
+
+    :returns: True if debian based, false otherwise.
+    :rtype: bool
+    .. note:: This is defined by looking at the python path
+       Currently I have not find a better way to find the correct installation
+       path in such case.
+
+       From : https://wiki.debian.org/Python
+       distutils setup scripts install files in /usr/local/ not sys.prefix
+       (which is normally /usr/). This is because /usr/ is reserved for files
+       installed from Debian packages. Note that
```

Comment:
This is exactly the same for rpm based distros ! I don't understand why we make a special case here.